index.js

```diff
@@ -46,6 +56,9 @@ controller.hears(['[\s\S]*'],['direct_message','direct_mention','mention'], func
   // Ping
   if (text === 'ping') return bot.reply('pong');
 
+  // Shortcut for `help(this)`
```

Comment:
Very true, both in the concept and in the naming of my feature. I can certainly make the necessary name changes and place my implementation in the more suitable `sandbox` property.